packages/remix-node/globals.ts

```diff
@@ -26,6 +28,10 @@ declare global {
 export function installGlobals({
   nativeFetch,
 }: { nativeFetch?: boolean } = {}) {
+  if (process.versions.deno) {
+    return;
+  }
```

Comment:
making this a no-op allows the @remix-run/serve cli to work without issues in Deno